system/include/webgpu/webgpu.h

```diff
@@ -54,6 +54,10 @@
 
```

Comment:
Is this file a verbatim copy from somewhere else?
 
If so can we add a comment at the top or maybe a README.md in the `system/include/webgpu` directory about where it comes from and the revision we are tracking?

Reply:
Done, although I didn't include a revision because not all of the Dawn changes have yet landed that allow generating these verbatim.